src/main/java/APIWorker.java

```diff
@@ -0,0 +1,50 @@
+import com.google.gson.*;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
```

Comment:
It might be better if APIWorker is a static class that implements the functions that we need. Then we just pass the course code that we need to the functions. Seems cleaner then having a different APIWorker for each class that we want to get information for.